_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,468 @@
+    }
+
+    .icon-inline {
+        display: inline-block;
+        margin-right: ($baseline / 4);
+        vertical-align: middle;
+    }
+}
+
+%button-primary {
+    @extend %button;
+    background-color: $brand-primary;
+
+    &:hover,
+    &:active,
+    &:focus {
+        background-color: $brand-primary-lighter;
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+%button-secondary {
+    @extend %button;
+    background-color: $brand-secondary;
+
+    &:hover,
+    &:active,
+    &:focus {
+        background-color: $brand-secondary-lighter;
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+%button-tertiary {
+    @extend %button;
+    background-color: $white;
+    border-color: $gray-darker;
+
+    &:hover,
+    &:active,
+    &:focus {
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+
+/* ============================
+14. Wells
+---------------------------- */
+%well {
+    box-shadow: inset 0 1px 2px 1px $shadow;
+    padding: ($baseline * 0.75) $baseline;
+}
+
+%no-content {
+    @extend %well;
+    padding: ($baseline*2);
+    background-color: $gray-lightest;
+    text-align: center;
+    color: $gray;
+
+    .button,
+    .action {
+        @extend %button-secondary;
+        margin-left: $baseline;
+
+        .icon {
+            margin-right: ($baseline / 2);
+        }
+    }
+}
+
+.is-loading {
+    @include animation(fadeIn $timing-fast linear 1);
+    @extend %well;
+    padding: ($baseline * 1.5) $baseline;
+    background-color: $white;
+    font-size: %font-size;
+    text-align: center;
+    opacity: .6;
+
+    .spin {
+        @extend %anim-rotateCW;
+        display: inline-block;
+    }
+
+    .copy {
+        padding-left: ($baseline / 4);
+    }
+}
+
+.is-hidden {
+    display: none;
+}
+
+
+/* ============================
+15. Lists
+---------------------------- */
+%list-unstyled {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-indent: 0;
+
+    li,
+    dt,
+    dd {
+        margin: 0;
+        padding: 0;
+    }
+}
+
+%list-inline {
+    @extend %list-unstyled;
+
+    li,
+    dt,
+    dd {
+        display: inline-block;
+    }
+}
+
+
+/* ============================
+16. Icon fonts
+---------------------------- */
+%icon-font {
+    display: inline-block;
+    min-width: 1rem; // Should be 16px
+    min-height: 1rem; // Should be 16px
+    speak: none;
```

Comment:
It's really for aural browsers and generally does nothing for visual browsers. But in basic terms, you can define how things are spoken (none, auto, inherit, spell-out). I wouldn't rely on it, but it can't hurt to include it. I'm okay omitting it if it just adds clutter.